app/javascript/packs/webauthn-unhide.js

```diff
@@ -0,0 +1,17 @@
+function unhideWebauthn() {
+  if (navigator && navigator.credentials && navigator.credentials.create) {
+    const elem = document.getElementById('select_webauthn');
+    if (elem) {
+      elem.classList.remove('hidden');
+    }
+  } else {
+    const checkboxes = document.querySelectorAll('input[name="two_factor_options_form[selection]"]');
+    for (let i = 0, len = checkboxes.length; i < len; i += 1) {
+      if (!checkboxes[i].classList.contains('hidden')) {
+        checkboxes[i].checked = true;
```

Comment:
Why are we setting `checked` to `true`? I'm not seeing a reference elsewhere to this property. Since we're talking radio buttons here, this should leave the last one selected, but we mark the first one selected by default when building the page.

Reply:
On the condition that webauthn is the only option, not doing this will result in webauthn checked and hidden as it is first in the list.  Resulting in a personal key option that is not selected yet is the only option.